src/debugging/python/PythonDebugHelper.ts

```diff
@@ -114,21 +72,14 @@ export class PythonDebugHelper implements DebugHelper {
                 },
                 true);
 
-        // These properties are required by the old debugger, should be changed to normal properties in the configuration
-        // as soon as the new debugger is released to 100% of the users.
-        const debugOptions = ['FixFilePathCase', 'RedirectOutput', 'ShowReturnValue'];
-
-        if (os.platform() === 'win32') {
-            debugOptions.push('WindowsClient');
-        }
+        const args = [...debugConfiguration.args || pythonRunTaskOptions.args || [], containerName];
+        const launcherPath = path.join(ext.context.asAbsolutePath('resources'), 'python', 'launcher.py');
 
         return {
-            ...debugConfiguration,
```

Comment:
Any reason it needs to be, "copy in a bunch of individual things from `debugConfiguration`", instead of the original `...debugConfiguration`?

Reply:
Yeah, the launch configuration for Python has a property called "python", which conflicts with the one we have in our own configuration so had to go this way.